Appsilon/mbaza
# Jakub improve explorer performance 184

Closes #184 

### Description
- use react-window to virtualize huge list of observations, in order to improve loading performance
- add new typescript's typings
- fade out scrolled card just a bit
- add loader instead of observation photo, when the card is being loaded

### How to test
- run the app and test observations panel manually
- check why installed types for react-window-infinite-loader are not detected automatically
